src/connection.js

```diff
@@ -138,9 +138,25 @@ export class StableWSConnection {
 			const { ws } = this;
 			isClosedPromise = new Promise(resolve => {
 				ws.onclose = () => {
+					this.logger(
+						'info',
+						`connection:disconnect() - resolving isClosedPromise`,
```

Comment:
first 2 logs aren't consistent with the rest.

Reply:
which once? If you can point me to exact log please...

Reply:
Maybe I am familiar with it since I see only the diff.

This one, see separator `-` vs `:`

Reply:
You are right. Fixed it now :)